book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure the `rentals` are also in the `setter` 
```rb
attr_accessor
```
As it can be modified and not read-only